src/manager/network-manager.c

```diff
@@ -1249,6 +1249,76 @@ int manager_configure_link_address(const IfNameIndex *ifidx,
         return dbus_network_reload();
 }
 
+int manager_replace_link_gateways(const IfNameIndex *ifidx, char **many, AddressFamily family) {
```

Comment:
It's a duplicate function. Use manager_configure_default_gateway_full

Reply:
> It's a duplicate function. Use manager_configure_default_gateway_full

I checked that manager_configure_default_gateway_full but not applicable it is only able to reset single gateway also if file has multiple gateway it is not able to remove  them instead of replacing them with new address.

Using sey-ipv46 if a user is provided only dhcp yes without specifying the address and gateway it should remove existing ipv6 addr and gateway.

Hence creating this function also manager_configure_default_gateway_full is used by other API's so modifying that function is not a good option.

Reply:
There can be only one default ipv4 and one ipv6 default GW. can you explain multiple ipv6 GW.

Reply:
> There can be only one default ipv4 and one ipv6 default GW. can you explain multiple ipv6 GW.
> 
> > Using sey-ipv46 if a user is provided only dhcp yes without specifying the address and gateway it should remove existing ipv6 addr and gateway.
> 
> you supply `keep=no` that would create a new file and automatically resets everything. We use `keep` for that pupose. Hence use it as other ctl commands

yes true, But expectation is it should reset only ipv6 addresses and gateway instead of creating a new file.

Reply:
manager_set_gateway should do it address looks OK